src/StatusBar.ts

```diff
@@ -240,6 +246,22 @@ export class StatusBar {
         return status
     }
   }
+  private getModes(modes?: Mode[]) {
+    if (!modes) {
+      return ''
+    }
+    const modesStrings = modes.map(m => {
+      switch (m) {
+        case 'coverage':
+          return '$(color-mode)'
+        case 'watch':
+          return '$(eye)'
+        default:
+          throw new Error(`unrecognized mode: ${m}`)
+      }
+    })
+    return modesStrings.length > 0 ? `${modesStrings.join(' ')}` : ''

```

Comment:
Why not simply `return modesStrings.join(' ')`?

Reply:
indeed